bbconf/config_parser/bedbaseconfig.py

```diff
@@ -205,6 +219,35 @@ def _init_qdrant_backend(self) -> QdrantBackend:
                 f"error in Connection to qdrant! skipping... Error: {err}", UserWarning
             )
 
+    def _init_qdrant_text_backend(self) -> QdrantBackend:
+        """
+        Create qdrant client text embedding object using credentials provided in config file
+
+        :return: QdrantClient
+        """
+
+        return QdrantBackend(
+            dim=384,
```

Comment:
this just looks funny. why is it this specific number?

Reply:
@nleroy917

Reply:
Thats the dimensionality of the embeddings for `sentence-transformers/all-MiniLM-L6-v2`. So its fine **iff** the user will never be able to parametrize this particular variable.

Regardless, I agree it should be annotated with a comment or a constant that is-self-descriptive (i.e. `EMBEDDING_DIMENSION` to help explain whats going on)